mbed_flasher/flashers/FlasherMbed.py

```diff
@@ -239,34 +223,79 @@ def try_pyocd_flash(self, source, target):
                               err, target["target_id"])
             return EXIT_CODE_FLASH_FAILED
 
+    def try_drag_and_drop_flash(self, source, target, no_reset):
+        """
+        Try to flash the target using drag and drop method.
+        :param source: file to be flashed
+        :param target: target board to be flashed
+        :param no_reset: whether to reset the board after flash
+        :return: 0 if success
+        """
+
+        target = FlasherMbed.refresh_target(target["target_id"])
+        if not target:
+            return EXIT_CODE_TARGET_ID_MISSING
+
+        mount_point = os.path.abspath(target['mount_point'])
+        (_, tail) = os.path.split(os.path.abspath(source))
+        destination = abspath(join(mount_point, tail))
+
+        try:
+            if 'serial_port' in target and not no_reset:
+                self.reset_board(target['serial_port'])
+                sleep(0.1)
+
+            copy_file_success = self.copy_file(source, destination)
+            if copy_file_success == EXIT_CODE_FILE_COULD_NOT_BE_READ:
+                return EXIT_CODE_FILE_COULD_NOT_BE_READ
+
+            self.logger.debug("copy finished")
+            sleep(4)
```

Comment:
why this hard-coded delay? not good..

Reply:
Not in the scope of this PR and also other PR open for this (#83).